.circleci/config.yml

```diff
@@ -67,6 +96,7 @@ workflows:
   build-test:
     jobs:
       - runtest
+      - deployment-checks
```

Comment:
I didn't add a filter for this yet. The job runs pretty fast right now, so my thinking was to leave it on all commits until we have the need to limit it if stuff is slow. Not a strong preference, though.

Reply:
update: they do take long. I'm adding the filter here as well.